package.json

```
@@ -70,12 +70,14 @@
 		"compile": "yarn run webpack --config source/webpack.prod.js && yarn run webpack --config source/webpack.prod.legacyBrowser.js",
 		"simple-compile": "yarn run webpack --config source/webpack.prod.js",
 		"stats": "webpack --config source/webpack.prod.js --profile --json > stats.json",
-		"eslint-check": "eslint --print-config .eslintrc | eslint-config-prettier-check",
-		"eslint": "LIST=`git diff --cached --name-only --diff-filter=AMR HEAD | grep .*\\.js | grep -v json`; if [ \"$LIST\" ]; then eslint $LIST; fi",
+		"eslintrc-check": "eslint --print-config .eslintrc | eslint-config-prettier-check",
+    "eslint-check": "eslint source test cypress",
+    "prettier-check": "prettier --check \"{source,test,cypress}/**/*.{js,jsx,ts}\"",
```

```suggestion
    "prettier-check": "prettier --check \"{source,test,cypress}/**/*.{js,jsx,ts,tsx}\"",
```

Une commande indépendante `prettier-check` est nécessaire ? Eslint ne peut pas l'intégrer ?